test/e2e/test_cluster_configs/no_outbound.json

```diff
@@ -19,7 +19,15 @@
 					"name": "agentpool1",
 					"count": 1,
 					"vmSize": "Standard_D2_v3",
-					"availabilityProfile": "VirtualMachineScaleSets"
+					"availabilityProfile": "VirtualMachineScaleSets",
+					"distro": "aks-ubuntu-16.04"
+				},
+				{
```

Comment:
I noticed we weren't validating the 18.04-LTS VHD in the "no outbound" test config. @marosset can you confirm that comprehensive Windows VHD coverage is in this config?

cc @mboersma

Reply:
We currently only produce Windows VHDs for Windows Server 2019 which will get exercised by this config so it is as this is as comprehensive as we can make it at the momemt.